### PR TITLE
FB - Web Hosting Multisites - Adding Diagnostic part for domains

### PR DESCRIPTION
**Pull request by fabienOVH**
Waiting for the "GO" of @GuiF008 after his review

### CODE REVIEW

**Comment by GuiF008** [comment]
go !

File: pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-green-info.png
Comment (Kazarus4) [inline pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-green-info.png:1]: Hi Fabien, the IP 192.0.2.0 is dedicated to show examples with local IPs into documentation. Could-you use the RFC IP 203.0.113.0 plz ?

File: pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-grey-popup.png
Comment (Kazarus4) [inline pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-grey-popup.png:1]: Hi Fabien, the IP 192.0.2.0 is dedicated to show examples with local IPs into documentation. Could-you use the RFC IP 203.0.113.0 plz ?

File: pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-yellow-info.png
Comment (Kazarus4) [inline pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-yellow-info.png:1]: Hi Fabien, the IP 192.0.2.0 is dedicated to show examples with local IPs into documentation. Could-you use the RFC IP 203.0.113.0 plz ?

File: pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-yellow-popup.png
Comment (Kazarus4) [inline pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-yellow-popup.png:1]: Hi Fabien, the IP 192.0.2.0 is dedicated to show examples with local IPs into documentation. Could-you use the RFC IP 203.0.113.0 plz ?

File: pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md [hunk 1]
```diff
@@ -153,6 +154,42 @@ Notez les éléments qui s’affichent, puis cliquez sur le bouton `Valider`{.ac
 > Les entrées DNS de type **A** et **TXT** doivent obligatoirement être placées dans la zone DNS active de votre nom de domaine pour qu'il soit ajouté à votre hébergement web. Seuls les entrées DNS de type **AAAA** sont optionnelles. 
 >
 
+#### Étape 2.3 : diagnostiquer vos noms de domaine <a name="diagnostic-domain"></a>
+
+Dans le tableau présent dans l'onglet `Multisite`, une colonne `Diagnostic` vous informe si votre nom de domaine pointe correctement vers l'hébergement associé. Elle permet de vérifier rapidement que la configuration DNS est bien alignée avec votre service d'hébergement, vous aidant ainsi à identifier et résoudre les problèmes de pointage éventuels. Pour chaque nom de domaine, trois valeurs de diagnostic sont possibles :
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md]:
```suggestion
Dans le tableau présent dans l'onglet `Multisite`, une colonne `Diagnostic` vous informe si votre nom de domaine pointe correctement vers l'hébergement web associé. Elle permet de vérifier rapidement que la configuration DNS de votre nom de domaine est correctement effectuée avec votre hébergement web. Ainsi, cette colonne vous aide à identifier et résoudre d'éventuels problèmes de pointage. Pour chaque nom de domaine, trois résultats de diagnostic sont possibles :
```

File: pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md [hunk 2]
```diff
@@ -153,6 +154,42 @@ Notez les éléments qui s’affichent, puis cliquez sur le bouton `Valider`{.ac
...
+
+- `A/AAAA` vert
+- `A/AAAA` jaune
+- `A/AAAA` gris
+
+##### A/AAAA vert
+
+![multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-green-info.png){.thumbnail}
+
+Lorsque l'icône `A/AAAA` est verte dans la colonne `Diagnostic`, cela signifie que l'enregistrement **A** (pour les adresses IPv4) et/ou l'enregistrement **AAAA** (pour les adresses IPv6) de votre nom de domaine pointe correctement vers l'adresse IP de votre hébergement. Cela indique que la configuration DNS est conforme et que votre nom de domaine est bien relié à l'hébergement prévu.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md]:
```suggestion
Lorsque l'icône `A/AAAA` est verte dans la colonne `Diagnostic`, cela signifie que l'enregistrement **A** (pour les adresses IPv4) et/ou l'enregistrement **AAAA** (pour les adresses IPv6) de votre nom de domaine pointe correctement vers l'adresse IP de votre hébergement web. La configuration DNS de votre nom de domaine est donc conforme pour fonctionner avec votre hébergement web.
```

File: pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md [hunk 3]
```diff
@@ -153,6 +154,42 @@ Notez les éléments qui s’affichent, puis cliquez sur le bouton `Valider`{.ac
...
+
+##### A/AAAA jaune
+
+![multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-yellow-info.png){.thumbnail}
+
+Lorsque l'icône `A/AAAA` est jaune dans la colonne `Diagnostic`, cela signifie que l'enregistrement **A** (IPv4) et/ou **AAAA** (IPv6) de votre nom de domaine pointe vers une addresse IP, mais qu'il ne s'agit pas de celle de l'hébergement associé.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md]:
```suggestion
Lorsque l'icône `A/AAAA` est jaune dans la colonne `Diagnostic`, cela signifie que l'enregistrement **A** (IPv4) et/ou **AAAA** (IPv6) de votre nom de domaine pointe vers une addresse IP, mais qu'il ne s'agit pas de celle de l'hébergement web depuis lequel vous consultez la colonne `Diagnostic`.
```

File: pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md [hunk 4]
```diff
@@ -153,6 +154,42 @@ Notez les éléments qui s’affichent, puis cliquez sur le bouton `Valider`{.ac
...
+Cliquez sur l'icône `A/AAAA` jaune pour plus d'informations. Le message suivant s'affiche :
+
+![multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-yellow-popup.png){.thumbnail}
+
+Pour résoudre les problèmes de pointage de votre nom de domaine et vous assurer qu'il pointe correctement vers l'hébergement souhaité, suivez les étapes décrites dans notre guide [« Éditer une zone DNS OVHcloud »](/pages/web_cloud/domains/dns_zone_edit){.external}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md]:
```suggestion
Pour résoudre les problèmes de pointage DNS de votre nom de domaine et vous assurer qu'il pointe correctement vers l'hébergement web souhaité, suivez les étapes décrites dans notre guide « [Éditer une zone DNS OVHcloud](/pages/web_cloud/domains/dns_zone_edit) ».
```

File: pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md [hunk 5]
```diff
@@ -153,6 +154,42 @@ Notez les éléments qui s’affichent, puis cliquez sur le bouton `Valider`{.ac
...
+##### A/AAAA gris
+
+![multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-grey-info.png){.thumbnail}
+
+Lorsque l'icône `A/AAAA` est grise dans la colonne `Diagnostic`, cela signifie que le nom de domaine ne pointe actuellement vers aucune adresse IP et qu'aucun enregistrement **A** ou **AAAA** n'est configuré pour ce nom de domaine.
+Cliquez sur l'icône `A/AAAA` grise pour plus d'informations. Le message suivant s'affiche :
+
+![multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/diagnostic-grey-popup.png){.thumbnail}
+
+Pour ajouter les enregistrements **A** ou **AAAA** et configurer correctement votre domaine, suivez les étapes décrites dans notre guide [« Éditer une zone DNS OVHcloud »](/pages/web_cloud/domains/dns_zone_edit){.external}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/multisites_configure_multisite/guide.fr-fr.md]:
```suggestion
Pour ajouter les enregistrements **A** ou **AAAA** et configurer correctement votre nom de domaine, suivez les étapes décrites dans notre guide « [Éditer une zone DNS OVHcloud](/pages/web_cloud/domains/dns_zone_edit) ».
```

**Review by Y0Coss — APPROVED** [review]

**Comment by Y0Coss** [comment]
FR is good. Please provide us with the EN version as well.

**Comment by fabienOVH** [comment]
> FR is good. Please provide us with the EN version as well.

en-gb version done

**Comment by Y0Coss** [comment]
Thank you for your contribution @fabienOVH

**Pull request merged**